src/components/screen/MemoList/MemoList.tsx

```diff
@@ -0,0 +1,23 @@
+import { MemoListProps } from './MemoList.types'
+
+const MemoList: React.FC<MemoListProps> = ({ users }) => {
+  const filterUsers = users.filter((user) => {
+    return user.memo != '' && user.memo
```

Comment:
user.memoがない場合の処理をしたいのなら、user.memoがtruthyなものを見るだけで十分
あと比較演算子は`!=`ではなく`!==`として型を厳密にしよう
```suggestion
    return user.memo
```